spec/facter/resolvers/augeas_resolver_spec.rb

```diff
@@ -7,12 +7,69 @@
 
   before do
     augeas.instance_variable_set(:@log, log_spy)
-    allow(Facter::Core::Execution).to receive(:execute)
-      .with('augparse --version 2>&1', logger: log_spy)
-      .and_return('augparse 1.12.0 <http://augeas.net/>')
+    allow(Facter::Resolvers::Augeas).to receive(:require).with('augeas')
```

Comment:
do you still need this if you mocked the require from kernel?

Reply:
You are right, this is no longer needed.